napari/components/experimental/chunk/_loader.py

```diff
@@ -46,16 +47,20 @@ class ChunkLoader:
     ----------
     synchronous : bool
         If True all requests are loaded synchronously.
+    num_workers : int
+        The number of worker threads or processes.
```

Comment:
```suggestion
        The number of worker threads.
```
We just support processes at this point right? Or can it be either?